wiki/Game_mode/de.md

```diff
@@ -1,3 +1,9 @@
+---
+stub: true
+tags:
+  - outdated
+---
```

Comment:
This is not the correct way of marking translations as outdated, see https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#outdated-translations.

Reply:
fixed (I think, im not 100% sure so ill leave this open for now)

Reply:
Done